### DISCUSSION TITLE
I can't find my Codespaces...

### DISCUSSION THREAD

**Discussion by David-Arcos-369** [community/community#30460] [Codespaces]
I can't find my Codespaces for testing environments?
It appears only under the sign in dropdown menu "Your codespaces"  but it doesn't show under the green button "Code" dropdown menu, all I have is Open with Github Desktop and Download Zip.
I also created an organization with one member, and a new repository under the organization, but I don't seem to be able to use Codespaces, why?

**Comment by delosemre** [comment]
Hello, I would like you to take a look at the link here. I hope it helps.
https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/enabling-github-codespaces-for-your-organization
